### PR TITLE
Add deploy ci

### PR DESCRIPTION
**Pull request by 8W9aG**

### CODE REVIEW

**Comment by 8W9aG** [comment]
@dflems @jgavris @rastersize

File: .travis.yml [hunk 1]
```diff
@@ -39,3 +39,12 @@ before_install: ./ci/before_install.sh
...
 script: ./ci/script.sh
 after_success: ./ci/after_success.sh
+deploy:
+  provider: releases
+  api_key:
+    secure: ANvlZh4vgRuTeUE9IenqsiKU0iqHVVJGDmqjvqmg0PMcN6QgR7leAAf45cQvPwKeJqS7DeXHFLq7gs0c7gfofBluFpt/UdCcHb1L7aXG+14uPVp6f5ggK5ZUxXwWfkW4s1xtAvDvQmGo7qB9Q/EaUSHJvH2vOBPlaaFc+h6C7M4=
+  file:
+    SPTDataLoader.framework.zip
+  on:
+    repo: spotify/SPTDataLoader
```
Comment (jgavris) [inline .travis.yml]: `tags: true` or `branch: master`?

Comment (rastersize) [inline .travis.yml]: I’d say `tags: true`.

As we should only make binary releases for our tags IMO.

File: Gemfile [hunk 2]
```diff
@@ -6,3 +6,4 @@ gem 'slather',
   :branch   => "feature-profdata"
 gem 'xcpretty'
 gem 'xcpretty-travis-formatter'
+gem 'carthage'
```
Comment (rastersize) [inline Gemfile]: Don’t think there’s a carthage gem? You install it via Homebrew. Also Travis-CI has it installed by default.

We might want to make sure it’s the latest though, so in our `ci/before_install.sh` script add:
```shell
brew update
brew install carthage
```

Comment (jgavris) [inline Gemfile]: I'd probably move it into `deploy.sh` as that's the only consumer.

Comment (jgavris) [inline Gemfile]: You need to check in any changes to a gemfile with the gemfile.lock. I'd just install this gem afterwards in `deploy.sh` if all goes well prior to that step.

File: .travis.yml (hunk 1 above)
Comment (8W9aG) [inline .travis.yml]: Yeh will make it tags: true

File: Gemfile (hunk 2 above)
Comment (8W9aG) [inline Gemfile]: Will move into deploy.sh

**Comment by jgavris** [comment]
:+1:

**Comment by rastersize** [comment]
:tada: :ship: :it:

**Pull request merged**